src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1600,13 +1600,13 @@ public static IEnumerable<IEdmStructuralProperty> Key(this IEdmEntityType type)
         /// <summary>
         /// Gets the declared alternate keys of the most defined entity with a declared key present.
         /// </summary>
-        /// <param name="type">Reference to the calling object.</param>
         /// <param name="model">The model to be used.</param>
+        /// <param name="type">Reference to the calling object.</param>
         /// <returns>Alternate Keys of this type.</returns>
-        public static IEnumerable<IDictionary<string, IEdmProperty>> DeclaredAlternateKeys(this IEdmEntityType type, IEdmModel model)
+        public static IEnumerable<IDictionary<string, IEdmProperty>> GetAlternateKeysAnnotation(this IEdmModel model, IEdmEntityType type)

```

Comment:
Do alternate keys in the `IDictionary<string, IEdmProperty>` have an order? if yes, please use an ordered data structure, such as `IOrderedDictionary`.